unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Tests/EventsSubSectionComponentControllerTests.cs

```diff
@@ -128,7 +124,7 @@ public void LoadFeaturedEventsCorrectly()
         eventsSubSectionComponentController.eventsFromAPI = ExploreEventsTestHelpers.CreateTestEventsFromApi(numberOfEvents);
 
         // Act
-        eventsSubSectionComponentController.LoadFeaturedEvents();
+        eventsSubSectionComponentController.view.SetFeaturedEvents(PlacesAndEventsCardsFactory.CreateEventsCards(eventsSubSectionComponentController.FilterFeaturedEvents()));
```

Comment:
Can we use `eventsSubSectionComponentView` instead?

Reply:
This was just the nearest substitution for fixing the test. As for `EventsSubSectionComponentView`, it has its own set of tests in the `EventsSubSectionComponentViewTests` and it already contains such tests cases like `SetFeatured(Going/Live/Trending)EventsCorrectly`, but with a little bit different logic.

Reply:
replaced it here https://github.com/decentraland/unity-renderer/pull/3917/commits/baa351f80f1d6f88de1aeb31df1dde6ff1cd095a